infra/modules/azure_function_app/data.tf

```diff
@@ -13,11 +13,6 @@ data "azurerm_private_dns_zone" "storage_account_file" {
   resource_group_name = local.private_dns_zone.resource_group_name
 }
 
-data "azurerm_private_dns_zone" "storage_account_table" {
```

Comment:
why remove this?

Reply:
Because it never gets called, and the pre-commit `tflint` failed